data-es6.js

```diff
@@ -1689,6 +1691,7 @@ exports.tests = [
         chrome49:    true,
         webkit:      true,
         edge12:      flag,
+		edge14:      true,
```

Comment:
if you could just globally replace any tabs that are in there in this PR, that would be great :-) thanks!